README.md

```diff
@@ -0,0 +1,40 @@
+# spine-doc-generator
+
+The command-line tool for Web Components documentation generating.
+[Polymer-analyzer](https://github.com/Polymer/polymer-analyzer) is used under the hood.
+
+## Installation
+```bash
+npm install -g git://github.com/SpineElements/spine-doc-generator.git
+```
+
+## Example
```

Comment:
Since these are more like the instructions, better rename this section to something like "Usage"